### ISSUE TITLE
Is there a way to use a .js file exporting json as a model?

### ISSUE THREAD

**Issue by jhyland87** [jaichandra/metalsmith-models#1]
I just started using metalsmith, and it looks like it checks for the `modeName + '.json'`, and I was wondering, if theres a way to allow .js files as well? (If they export json)

Thanks!

**Comment by jaichandra** [comment]
Right now, there is no such support. Can be a good addition though.

**Comment by jhyland87** [comment]
Yeah, I definitely agree. It couldnt contain any asynchronous behavior though, since it would need to return a JSON object without using a callback or promises.
